lib/moon_web/pages/design/tabs_page.ex

```diff
@@ -0,0 +1,347 @@
+    ]
+  )
+
+  data(props_info_array_tab, :list,
+    default: [
+      %{
+        :name => 'default',
+        :type => 'slot ',
+        :required => 'Yes',
+        :default => '-',
+        :description => 'Content inside the tab'
+      },
+      %{
+        :name => 'class',
+        :type => 'css_class',
+        :required => '-',
+        :default => '-',
+        :description => 'additional css classes for the tab'
+      },
+      %{
+        :name => 'disabled',
+        :type => 'boolean',
+        :required => 'No',
+        :default => 'false',
+        :description => 'Is tab disabled'
+      }
+    ]
+  )
+
+  data(props_info_array_tabs, :list,
+    default: [
+      %{
+        :name => 'tabs',
+        :type => 'slot ',
+        :required => 'No',
+        :default => '-',
+        :description => 'List of the tabs. Reffer to Moon.Design.Tabs.Tab module'
+      },
+      %{
+        :name => 'panels',
+        :type => 'slot ',
+        :required => 'No',
+        :default => '-',
+        :description => 'Panels slot. Panel with selected_index is shown'
+      },
+      %{
+        :name => 'on_change',
+        :type => 'event',
+        :required => 'No',
+        :default => '-',
+        :description => 'Event to happen when non-disabled tab is clicked'
+      },
+      %{
+        :name => 'selected_index',
+        :type => 'integer',
+        :required => 'No',
+        :default => '0',
+        :description => 'Index of the selected tab'
+      },
+      %{
+        :name => 'size',
+        :type => 'sm | md',
+        :required => 'No',
+        :default => 'md',
+        :description => 'Size of tabs'
+      },
+      %{
+        :name => 'tab_titles',
+        :type => 'list',
+        :required => 'No',
+        :default => '[]',
+        :description =>
+          'List of tab titles. THe are rendered with tab_module. Only if no slot tabs assigned'
+      },
+      %{
+        :name => 'tab_module',
+        :type => 'atom',
+        :required => 'No',
+        :default => 'Moon.Design.Tabs.Tab',
+        :description => 'Module for rendering tabs from tab_titles'
+      }
+    ]
+  )
+
+  data(tab_id1, :integer, default: 0)
+  data(tab_id2, :integer, default: 0)
+  data(tab_id3, :integer, default: 0)
+  data(tab_id4, :integer, default: 0)
+  data(tab_id5, :integer, default: 0)
+  data(tab_id6, :integer, default: 0)
+  data(tab_id7, :integer, default: 0)
+  data(tab_id8, :integer, default: 0)
+
+  def anatomy_code() do
+    """
+    <Tabs selected_index={@tab_id} on_change="clicked_tab">
+      <Tabs.Tab>First tab</Tabs.Tab>
+      <Tabs.Tab>Second tab</Tabs.Tab>
+      <Tabs.Tab>Third tab</Tabs.Tab>
+      <Tabs.Panel>{lorem()}</Tabs.Panel>
+      <Tabs.Panel>{ipsum()}</Tabs.Panel>
+      <Tabs.Panel>{dolor()}</Tabs.Panel>
+    </Tabs>
+    """
+  end
+
+  def render(assigns) do
+    ~F"""
+    <Page {=@theme_name} {=@active_page} {=@breadcrumbs} {=@direction}>
+      <ComponentPageDescription title="Tabs">
```

Comment:
Should we add "In progress" badge at this point for all v2 components, indicating that breaking changes may occur, as practice on React side?